### ISSUE TITLE
Does it have a Pytorch only version?

### ISSUE THREAD

**Issue by wetliu** [mees/calvin#77]
Hello. Thank you for the collecting the datasets. This is awesome. 

May I ask if there is an Pytorch only implementation (at least for the dataset usage)? 

(Just my own opinion) I feel like as a dataset, using Torch Lightning is very confusing and the configurations are very hard to follow and track (wrapping in the conf folder with many different config files). The dataset itself should be lightweight and simple to use. 

Thank you!

**Comment by mees** [comment]
The reason for using Lightning DataModule is specifically that it makes it much easier to understand how data preparation works, data splitting and transforms works, instead of normal PyTorch code, where these things are usually scattered across many files. If you want to understand how the Lightning DataModule works I suggest reading https://lightning.ai/docs/pytorch/stable/data/datamodule.html
The underlying dataset is torch DataLoader, so you can also use our DataModule without Lightning: https://lightning.ai/docs/pytorch/stable/data/datamodule.html#datamodules-without-lightning

**Comment by lukashermann** [comment]
I created a gist with a [minimal example](https://gist.github.com/lukashermann/22bb02d6d139be1b9b157508ecdf8e2f) of how to use the dataset without lightning and the nested hydra configs
